src/Microsoft.Net.Http.Headers/EntityTagHeaderValue.cs

```diff
@@ -112,6 +112,24 @@ public override int GetHashCode()
             return _tag.GetHashCode() ^ _isWeak.GetHashCode();
         }
 
+        public static bool CompareStrongly(EntityTagHeaderValue left, EntityTagHeaderValue right)
```

Comment:
Hmm I guess I'll go with the bool for now but we often use enums even when only two values are possible to make it more explicit and calls more readable. For example in `DeploymentParameters` for runtime flavour and runtime architecture. Otherwise we have to use named parameters everywhere like `new DeploymentParameters(..., useCoreClrRuntime: true, useX64Architecture: true)`